src/assets/styles.css

```diff
@@ -124,12 +124,16 @@ body {
 
 	& .tty-username {
 		color: #3465a4;
+		color: var(--lilac);
+		color: #06aeb1;
+		color: #d8b4dd;
```

Comment:
Why are there so many color properties here?

Reply:
Mostly so you can ask that question and see if you like them better :slightly_smiling_face: 

Ultimately if you're good with these values I want to remove them. If you check them they get successively lighter.

Reply:
Ah! Well, then... I choose #06aeb1 (line 128). The next value #d8b4dd is really darn close to "skyblue", which is used in the git diff.

Reply:
Are you sure you're looking at this right? The current `#d8b4dd` is  light purple shade but `#06aeb1` is a light blue that's fairly close to the git diff color (that's why I changed it)

Reply:
VSCode tells me that "skyblue" is `#87ceeb`.

Reply:
Right, I wanted the username and hostname in the prompt to be a separate color from the cyan like blue of the diff, so I picked light purple.

Reply:
`#d8b4dd` is really darn close to `#87ceeb`. What am I missing?

There's almost no red in `#06aeb1`... Much larger difference to my eyes. Maybe increase the green value a little?

Reply:
PING!